earth_enterprise/src/common/packet.h

```diff
@@ -425,18 +427,18 @@ struct etPointerTranslator
 {
   etPointerCypher * pointerList;
   int pointerListSize;
-    
+
```

Comment:
unrelated: a big chunk of this code needs to be ported over to c++11, at some point we should think about creating an issue for this